docs/source/user/troubleshoot.rst

```diff
@@ -223,11 +223,174 @@ When this mode is enabled, the ``vectorize`` and ``guvectorize`` decorators will
 still result in compilation of a ufunc, as there is no straightforward pure
 Python implementation of these functions.
 
+
+Debugging JIT compiled code with GDB
+====================================
+
+Setting the ``debug`` keyword argument in the ``jit`` decorator
```

Comment:
I see, so the idea is that `debug=True` has multiple effects, and turning on debug information is only one of those effects.  In that case, what you have now makes sense.